### PR TITLE
Katello 3.13 test and release jobs

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.13.groovy [hunk 1]
```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent none
+
+    environment {
+        foreman_version = "1.22"
```
Comment (tbrisker) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.13.groovy]:
```suggestion
        foreman_version = "1.23"
```

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.13.groovy [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
```
Comment (tbrisker) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.13.groovy]: 2 hours is not always enough for katello, i would raise it to 3 (not only here, but i won't ask for the rest in this pr)

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.13.groovy [hunk 3]
```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent none
+
+    environment {
+        foreman_version = "1.23"
+    }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.13.groovy]:
```suggestion
        timeout(time: 3, unit: 'HOURS')
```

**Comment by evgeni** [comment]
> It'd be nice to template these as well but doesn't have to be in this PR.

Yeah, I thought the same, but then went full "I want it working in the next 10 minutes" mode :)

**Pull request merged**